nvflare/job_config/fed_job_config.py

```diff
@@ -172,7 +172,7 @@ def _copy_ext_scripts(self, custom_dir, ext_scripts):
                     relative_script = self._get_relative_script(script)
                 else:
                     relative_script = script
-                dest_file = os.path.join(custom_dir, relative_script)
+                dest_file = os.path.join(custom_dir, os.path.basename(relative_script))
```

Comment:
This change has nothing to do with ScriptExecutor. Please use a different PR.

Reply:
This is needed for better experience with script and task_script_path args of SubprocessLauncher and InProcessClientAPIExecutor